other_files/localai_installer/localai_installer.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+curl -sSL https://tea-cup.midori-ai.xyz/download/localai_installer_linux.tar.gz -o localai_installer.tar.gz
```

Comment:
The server does not have this file... im not sure how to make this file and it be uploaded to the file server? Some help would be lovely, once we have that set up ill take the PR.